SpatialGDK/Source/SpatialGDK/Public/SimulatedPlayers/SimulatedPlayer.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Kismet/BlueprintFunctionLibrary.h"
+#include "SimulatedPlayer.generated.h"
+
+UCLASS()
+class SPATIALGDK_API USimulatedPlayer : public UBlueprintFunctionLibrary
```

Comment:
Make this class name more reflective of what it does. e.g. `SimPlayerBPFunctionLibrary`